### PR TITLE
Fix phpdoc in FeatureEvent

### PR DESCRIPTION
**Pull request by Denitz**
## Summary of Changes

Invalid phpdoc

### Testing Instructions

Feature/unfeature article.

### Actual result BEFORE applying this Pull Request

Invalid phpdoc

### Expected result AFTER applying this Pull Request

Valid phpdoc

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [x] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_content/src/Event/Model/FeatureEvent.php [hunk 1]
```diff
@@ -71,9 +71,9 @@ public function __construct($name, array $arguments = [])
     }
 
     /**
-     * Set used parameter to true
+     * Abort with reason
```
Comment (richard67) [inline administrator/components/com_content/src/Event/Model/FeatureEvent.php]: To me the new comment reads as if the method would abort execution. But it does only set the "abort" parameter to true and sets the "reason" parameter. So I'm not sure if the new comment is ok. I'd like to ask a native speaker => @brianteeman Could you check if the proposed comment is ok?

Comment (laoneo) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: @brianteeman can you help out here?

Comment (brianteeman) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: sorry didnt see this before. Hard to comment as I dont understand it. Shouldnt line 76 actualy say what the "reason" is

Comment (richard67) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: My problem is that the "Abort with reason" reads as if that method would perform the abort, but it only sets a parameter "abort" and a reason. Of course this is still better than without this PR where the comment is absolutely wrong. But I wanted to ask if someone has a better idea for that doc block comment, e.g. "Set abort and reason parameters".

But if I am the only one having that problem I am happy with the PR as it is.

Comment (Quy) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: Reason is passed to the method. `Set abort and reason parameters.` 👍 

https://github.com/Denitz/joomla-cms/blob/6aa724f23b6914c471c213c486161b8a365ca62e/plugins/workflow/featuring/src/Extension/Featuring.php#L359

Comment (Quy) [inline administrator/components/com_content/src/Event/Model/FeatureEvent.php]:
```suggestion
     * Set abort and reason parameters.
```

Comment (Quy) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: or `Set abort with reason.`

Comment (Denitz) [reply administrator/components/com_content/src/Event/Model/FeatureEvent.php]: Done.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 46bb5b3385de18b13633c5fe4a596e144dc4149f<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42587">issues.joomla.org/tracker/joomla-cms/42587</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 46bb5b3385de18b13633c5fe4a596e144dc4149f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42587">issues.joomla.org/tracker/joomla-cms/42587</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42587">issues.joomla.org/tracker/joomla-cms/42587</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**